tools/tpm2_tool.c

```diff
@@ -61,18 +62,101 @@ static ESYS_CONTEXT* ctx_init(TSS2_TCTI_CONTEXT *tcti_ctx) {
     return esys_ctx;
 }
 
+/*
+ * Build a list of the TPM2 tools linked into this executable
+ */
+static const tpm2_tool ** tools;
```

Comment:
That doesn't work since it would need to have a static size (and the `calloc()` removed):
```
tools/tpm2_tool.c:74:15: error: ‘tools’ has an incomplete type ‘const tpm2_tool *[]’ {aka ‘const struct <anonymous> *[]’}
   74 |         tools = calloc(1024, sizeof(*tools));
```

Reply:
Can you just statically allocate it since the array size is static below, just heap allocated?